joomla/joomla-cms
# [4.0] Remove horizontal scrollbar in statistics modal

### Summary of Changes
Use `container-popup` to remove extra margins resulting in horizontal scrollbar. Make markup consistent with other modals.


### Testing Instructions
Go to Components > Smart Search
Click `Statistics` button
See horizontal scrollbar
Apply PR
No horizontal scrollbar

### Actual Results

![statistics-popup](https://user-images.githubusercontent.com/368084/67642940-54225b80-f8ce-11e9-9a67-f90f60f1b797.jpg)
